### PR TITLE
Irrlich 1.8.3

### PR DESCRIPTION
**Pull request by AdrianArroyoCalle**
https://codein.withgoogle.com/dashboard/task-instances/4889102981791744/?sp-page=1

### CODE REVIEW

File: dev-games/irrlicht/irrlicht-1.8.3.recipe [hunk 1]
```diff
@@ -0,0 +1,80 @@
+SUMMARY="Open source high performance realtime 3D engine written in C++ "
+DESCRIPTION="The Irrlicht Engine is an open source high performance realtime 3D engine \
+written in C++. It is completely cross-platform, using D3D, OpenGL and its \
+own software renderer, and has all of the state-of-the-art features which can \
+be found in commercial 3d engines. We've got a huge active community, and \
+there are lots of games in development that use the engine. You can find \
+enhancements for Irrlicht all over the web, like alternative terrain \
+renderers, portal renderers, world layers, tutorials, editors, bindings for \
+perl, ruby, python, and so on. And best of all: It's completely free"
```
Comment (waddlesplash) [inline dev-games/irrlicht/irrlicht-1.8.3.recipe]: End with period.

File: dev-games/irrlicht/irrlicht-1.8.3.recipe [hunk 2]
```diff
@@ -0,0 +1,80 @@
+SUMMARY="Open source high performance realtime 3D engine written in C++ "
```
Comment (waddlesplash) [inline dev-games/irrlicht/irrlicht-1.8.3.recipe]: Trim trailing space.

File: dev-games/irrlicht/irrlicht-1.8.3.recipe [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+DESCRIPTION="The Irrlicht Engine is an open source high performance realtime 3D engine \
+written in C++. It is completely cross-platform, using D3D, OpenGL and its \
```
Comment (waddlesplash) [inline dev-games/irrlicht/irrlicht-1.8.3.recipe]: Add an oxford comma, please :)

**Comment by waddlesplash** [comment]
LGTM other than comments.

File: dev-games/irrlicht/irrlicht-1.8.3.recipe [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+SOURCE_URI="http://downloads.sourceforge.net/irrlicht/irrlicht-1.8.3.zip"
+CHECKSUM_SHA256="9e7be44277bf2004d73580a8585e7bd3c9ce9a3c801691e4f4aed030ac68931c"
+PATCHES="irrlicht-1.8.3.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	irrlicht$secondaryArchSuffix = $portVersion
+	lib:libIrrlicht$secondaryArchSuffix
```
Comment (KapiX) [inline dev-games/irrlicht/irrlicht-1.8.3.recipe]: Version compat could be useful here.

**Comment by AdrianArroyoCalle** [comment]
Updated

**Pull request merged**